app/src/main/java/com/keylesspalace/tusky/db/TimelineDao.kt

```diff
@@ -86,18 +87,39 @@ WHERE timelineUserId = :accountId AND (serverId = :statusId OR reblogServerId =
     )
     abstract fun removeAllByUser(accountId: Long, userId: String)
 
+    /**
+     * Removes everything in the TimelineStatusEntity and TimelineAccountEntity tables for one user account
+     * @param accountId id of the account for which to clean tables
+     */
+    @Transaction
```

Comment:
We've been doing this before but now I realize that this transaction is *huge* and we don't even really care about it